packages/image_picker/image_picker_platform_interface/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 2.5.0
+
+* Re-adding `forceFullMetadata` option, but as a parameter of a new `pickImageFromSource` 
+  method (non-breaking change approach).
```

Comment:
You don't need to explain in the changelog that it's non-breaking; that's indicated by the version.

Reply:
Okay, I only wanted to explicitly show the difference from previous implementation